bids2openminds/main.py

```diff
@@ -12,6 +13,53 @@
 from .mapping import bids2openminds_instance
 
 
+def create_openminds_person(full_name):
+
+    alternate_names = []
+    person = HumanName(full_name)
+    given_name = person.first
+    family_name = person.last
+
+    if person.middle:
```

Comment:
We should discuss this in the openMINDS meeting. In the EBRAINS KG it is common to add middle initials to givenName.

Reply:
actually the preferred convention is to provide middle names in full not only initials (to avoid ambiguity)